assets/blank.html

```diff
@@ -1,7 +1,7 @@
 <html>
   <head>
-    <script src="https://unpkg.com/@microsoft/mgt@2/dist/bundle/mgt-loader.js"></script>
-    <mgt-msal2-provider client-id="6b5a42f1-9e1a-4a20-9df2-9b058df3c213"></mgt-msal2-provider>
+    <script src="https://unpkg.com/@microsoft/mgt@latest/dist/bundle/mgt-loader.js"></script>
```

Comment:
This will stop working when we publish v4

Reply:
Very true. We don't have an example for a auth page for pop-ups that is code-driven. Anything you could share to save me time?

Reply:
We could initialize the provider imperatively, 

I'll have a look at it later this morning.

Reply:
I actually fixed it. We'll have an auth.html page with minimal JS that uses our `CLIENTID`.